### ISSUE TITLE
MP Mini Input recognized too early

### ISSUE THREAD

**Issue by NicoHood** [limpkin/mooltipass#259]
### Expected behavior
If you have an information like "No Stored Favorites" and then scroll it should ignore the scrolls until the message is over.

### Actual behavior
The scrolling will take effect after the message was shown. But always only a single scrol is recognized.

### Step by step guide to reproduce the problem
Have no login or favorites stored and then enter this section. It will show an information and then scroll down. This also applies to many other menus such as if you change the pin and then abort etc.

### Mooltipass version
v0.7 (current master) but self compiled for MP Mini

**Comment by limpkin** [comment]
I disagree with the expected behavior. This behavior has been the same since the original mooltipass IIRC

**Comment by NicoHood** [comment]
I dont know if you missunderstand me but on the original mp there is no such a bug/behavior as the whole menu and input is structed different. However I fixed that bug in the PR.
